### PR TITLE
feat(ps) show power connection and battery warnings

### PR DESCRIPTION
**Pull request by carolinemodic**
Screenshots of new UI, all UI is modelled after BMD, the copy is the same as with BMD. 

Screen Shot 2021-09-13 at 2 56 08 PM
Screen Shot 2021-09-13 at 3 17 04 PM
Screen Shot 2021-09-13 at 3 16 28 PM

### CODE REVIEW

File: apps/precinct-scanner/src/screens/InsertBallotScreen.tsx [hunk 1]
```diff
@@ -34,10 +45,17 @@ export default InsertBallotScreen
 
 /* istanbul ignore next */
 export const ZeroBallotsScannedPreview = (): JSX.Element => {
-  return <InsertBallotScreen scannedBallotCount={0} />
+  return (
+    <InsertBallotScreen scannedBallotCount={0} showNoChargerWarning={false} />
```
Comment (eventualbuddha) [inline apps/precinct-scanner/src/screens/InsertBallotScreen.tsx:49]: Add a preview for when we _do_ show the warning?

File: apps/precinct-scanner/src/screens/PollsClosedScreen.tsx [hunk 2]
```diff
@@ -18,5 +29,5 @@ export default PollsClosedScreen
 
 /* istanbul ignore next */
 export const DefaultPreview = (): JSX.Element => {
-  return <PollsClosedScreen />
+  return <PollsClosedScreen showNoChargerWarning={false} />
```
Comment (eventualbuddha) [inline apps/precinct-scanner/src/screens/PollsClosedScreen.tsx:32]: Add a preview for when we _do_ show the warning?

File: apps/precinct-scanner/src/screens/SetupPowerPage.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+              machine.
+            </p>
+          </Prose>
+        </MainChild>
+      </Main>
+    </CenteredScreen>
+  )
+}
+
+export default SetupPowerPage
```
Comment (eventualbuddha) [inline apps/precinct-scanner/src/screens/SetupPowerPage.tsx:27]: Add a preview?

**Review by eventualbuddha — APPROVED** [review]
LGTM. Some previews would be a nice-to-have.

**Comment by carolinemodic** [comment]
Added Previews

**Comment by eventualbuddha** [comment]
> Added Previews

I don't see them…

**Comment by carolinemodic** [comment]
ughhh, thought I pushed but I didn't. my bad will make another PR with them

**Pull request merged**